lib/api/workloadendpoint.go

```diff
@@ -31,6 +31,10 @@ func (t WorkloadEndpoint) GetResourceMetadata() unversioned.ResourceMetadata {
 	return t.Metadata
 }
 
+func (t WorkloadEndpoint) String() string {
+	return fmt.Sprintf("WorkloadEndpoint(Node=%s, Orchestrator=%s, Workload=%s, Name=%s, ActiveInstanceID=%s)", t.Metadata.Node, t.Metadata.Orchestrator, t.Metadata.Workload, t.Metadata.Name, t.Metadata.ActiveInstanceID)
```

Comment:
I think we can remove the Active Instance ID, or perhaps only include it if it is set to something - it's used under very specific circumstances.